notebooks/3. Cars.ipynb

```diff
@@ -94,7 +96,7 @@
     "    bgt_road_polygons_csv += parsed_content\n",
     "\n",
     "# Write the csv\n",
```

Comment:
In this example we use one tile. If we use a sequence we should use a better file name